src/components/Media/_media.scss

```diff
@@ -27,23 +27,48 @@
 }
 
 .cc-media__caption {
-  @include grid-column(1, 7);
+  align-self: start;
+  // D7 override
+  border: 0;
+  border-left: 1px solid var(--colour-grey-70);
+  color: var(--colour-grey-70);
+  font-size: var(--body-xs);
+  margin-top: calc(2 * var(--space-unit));
+  padding: calc(0.25 * var(--space-unit)) 0 calc(0.25 * var(--space-unit)) calc(2 * var(--space-unit));
+ 
+  &:before {
+    content: 'Above';
+  }
+}
 
+// caption is moved to right hand side of media 
+.cc-media--wide__caption {
+  @include grid-column(1, 7);
+  
   @include mq(sm) {
     @include grid-column(2, 12);
   }
-
+  
   @include mq(md) {
     @include grid-column(10, 13);
+    margin-top: 0;
+
+    &:before {
+      content: 'Left';
+    }
   }
 }
 
 .cc-media__caption-detail {
   display: block;
+  font-family: var(--font-tertiary);
+  font-weight: bold;
 }
 
 .cc-media__credit {
   display: block;
+  font-family: var(--font-tertiary);
+  -webkit-font-smoothing: auto;
```

Comment:
How come?

Reply:
this font is very small and in courier it gets very feint with antialiasing